src/input.js

```diff
@@ -65,10 +65,6 @@ class Input extends Component {
     this.updateInputValue(event.target.value)
   }
 
-  reset = () => {
```

Comment:
Could you make sure that we're not calling `.reset()` on the input anywhere else? I think we may be...